<launch>
	<node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk" output="screen">
		<!-- node parameters -->
		<param name="serial_name" type="string" value="/dev/ttyTHS1"/>
		<param name="baud_rate" type="int" value="230400"/>
		<param name="app_id" type="int" value="1022384"/>
		<param name="app_version" type="int" value="1"/>
		<param name="app_bundle_id" type="string" value="Welcome to use dji-sdk"/>
		<param name="enc_key" type="string" value="e7bad64696529559318bb35d0a8c6050d3b88e791e1808cfe8f7802150ee6f0d"/>
		<param name="groundstation_enable" type="int" value="1"/>
	</node>

	<node pkg="dji_dispatcher" type="dji_sdk_client" name="dji_sdk_client" output="screen">
	</node>
	<node pkg="dji_sdk_read_cam" type="dji_sdk_read_cam" name="dji_sdk_read_cam" output="screen">
		<!-- node parameters -->
		<param name="gray_or_rgb" type="int" value="0"/>
		<param name="to_mobile" type="int" value="1"/>
	</node>
</launch>
